.travis.yml

```diff
@@ -3,22 +3,9 @@ language: node_js
 node_js: "10"
 matrix:
   include:
-    - osx_image: xcode7.3
-      env: DEVICE=9.3
-
-    - osx_image: xcode8.3
-      env: DEVICE=9.3
-    - osx_image: xcode8.3
-      env: DEVICE=10.3
-
-    - osx_image: xcode9.3
-      env: DEVICE=10.3
     - osx_image: xcode9.3
       env: DEVICE=11.3
 
-    # TODO: uncomment once Travis 9.4 VMs have iOS 11.3 SDK installed
-    # - osx_image: xcode9.4
-    #   env: DEVICE=11.3
     - osx_image: xcode9.4
```

Comment:
how about adding new environments instead?

Reply:
@mykola-mokhnach yeah when I get a chance I'll put in a new PR and port this stuff over to Azure.